brandonrhayes/minimizingVaccineWasteWebsite
# Download button

Made the download button download a zip file. In the future, depending on download size we may want to implement a 'downloading' pop-up.

To test:
1) click on the download button on the website
2) ensure 'msci343_blinkPrototype.zip' is downloaded
